tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@components/*": ["src/components/*"]
+      "@components/*": ["src/components/*"],
+      "@assets/*": ["src/assets/*"]
```

Comment:
added to avoid those long ../../../ on i18n pages that uses images. What do you think?

Reply:
Yup, I was going to suggest this before but wasn't 100% sure if the aliases worked within MD syntax for images. Good call!